index.html

```diff
@@ -11,25 +11,176 @@
 </head>
 
 <body>
-    <Header>
+    <header>
         <a class="logo" href="#">My Logo</a>
         <nav class="menu"><img src="images/Union.png" alt="menu"></nav>
-    </Header>
-    <section>
-        <article>
+    </header>
+    <section class="work-container">
+        <section class="intro-section">
             <h1>I’m Vishal <br>Glad to see <br>you!</h1>
             <p>I’m a software developer! I can help you build a product, feature or website Look through some of my
                 work and experience! If you like what you see and have a project you need coded, don’t hestiate to
                 contact me.</p>
             <h2>LET’S CONNECT</h2>
             <ul>
-                <li><img src="images/twitter.png" alt="twitter"></li>
-                <li><img src="images/Linkedin.png" alt="twitter"></li>
-                <li><img src="images/medium.png" alt="medium"></li>
-                <li><img src="images/github.png" alt="github"></li>
-                <li><img src="images/angellist.png" alt="angellist"></li>
+                <li class="social-list"><img src="images/twitter.png" alt="twitter"></li>
+                <li class="social-list"><img src="images/Linkedin.png" alt="twitter"></li>
+                <li class="social-list"><img src="images/medium.png" alt="medium"></li>
+                <li class="social-list"><img src="images/github.png" alt="github"></li>
+                <li class="social-list"><img src="images/angellist.png" alt="angellist"></li>
             </ul>
-        </article>
+        </section>
+
+        <div class="work-item">
+            <div class="work-box">
+                <div class="card-image">
+                    <img class="work-snapshot" src="images/SnapshotPortfolio.png" alt="snapshot">
+                </div>
+                <h2 class="card-title">Tonic</h2>
+                <div class="work-detail-box">
+                    <ul>
+                        <li class=" work-detail canopy">CANOPY</li>
+                        <li class="work-detail work-text">Back End Dev</li>
+                        <li class="work-detail work-text">2015</li>
+                    </ul>
+                </div>
+                <p class="work-description">A daily selection of privately personalized reads; no accounts or sign-ups
+                    required.</p>
+                <div>
+                    <div class="work-detail-box">
+                        <ul>
+                            <li class="chips">html</li>
+                            <li class="chips">css</li>
+                            <li class="chips">javascript</li>
+                        </ul>
+                    </div>
+                </div>
+                <div class="work-button"><a class="project-button" href="#">See Project</a></div>
+            </div>
+        </div>
+        <div class="work-item">
+            <div class="work-box">
```

Comment:
- [ ] I think it's better to group all the projects in a `<section>` tag with a grid display. this will make your page more semantically correct.